simdjson/csimdjson.pyx

```diff
@@ -382,6 +382,13 @@ cdef class ParsedJson:
             raise JSONDecodeError('Error iterating over document', '', 0)
 
         return self._items(iter, parsed_query)
+    
+    def items_from_parsed_query(self, parsed_query):
```

Comment:
Rather than a separate method, `items` should accept both a string or a 'parsed query'.

Reply:
Ok, sounds good for me. Do we want to use the same parameter for both variants and call the necessary path depending on the type?

I am going to change it after Christmas. I wish you happy holidays.

Reply:
Yes exactly :) Lets keep the API simple.
Hope you had a good holiday.

Reply:
Yes,thanks, hope, you too. I added the type check to ParsedJson.items. For the time being, I used isinstance. I read that PyString_Check and PyList_Check are faster. However, these functions we would need to call in cdef function and from the speed test it doesn't look like that the isinstance has a major impact on the performance.